src/api/python/_api.py

```
@@ -1120,22 +1123,24 @@ def get_reg_descs(
+        Args:
+            descs: NIXL list to serialize.
 
-    def get_serialized_descs(self, descs) -> bytes:
+        Returns:
+            Serialized descriptor list.
+        """
         return pickle.dumps(descs)
 
-    """
-    @brief Deserialize NIXL descriptor list.
+    def deserialize_descs(self, serialized_descs: bytes):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `deserialize_descs`

(ANN202)